e2e/support/helpers/e2e-misc-helpers.js

```diff
@@ -201,8 +206,9 @@ export function visitDashboard(dashboard_id, { params = {} } = {}) {
         url: `/dashboard/${dashboard_id}`,
         qs: params,
       });
-
-      cy.wait(aliases);
+      if (should_wait !== false) {
```

Comment:
If waiting is not needed we could use `cy.visit(`/dashboard/${id}`)` directly as done in other tests